### ISSUE TITLE
Implement a crossover of two sounds for the water levels in the glasses on screen one

### ISSUE THREAD

**Issue by Ashton-Morris** [phetsims/mean-share-and-balance#183]
I have added these sounds in our shared audio folder for this sim.

- MSaBGlassLevelCrossfadeSound1.mp3
- MSaBGlassLevelCrossfadeSound2.mp3

We intend to try a crossfade between these sounds and use them for the pitch of the individual cups. Cup 1 would be 100% MSaBGlassLevelCrossfadeSound1.mp3, Cup 7 would be 100% MSaBGlassLevelCrossfadeSound2.mp3 and all of the cups in between would play somewhere in the middle as the crossfade shifts between.

**Comment by jbphet** [comment]
The folder where the sounds reside can be found [here](https://o365coloradoedu.sharepoint.com/sites/PHYS-phet-pilot/PhET%20Files/Forms/AllItems.aspx?csf=1&web=1&e=jhc6X1&cid=83d3c58c%2D3b8d%2D4e50%2D8ea6%2D9748dc2b6762&RootFolder=%2Fsites%2FPHYS%2Dphet%2Dpilot%2FPhET%20Files%2FPhET%20Files%2FPhET%20Sim%20Design%2FMean%5F%20Share%20and%20Balance%20HTML5%2FSounds%20and%20Sonification%2FSound%20Files&FolderCTID=0x0120001AF96E22694A1A448A9649A49C4AC974).

**Comment by jbphet** [comment]
An initial version of the cross-mix has been implemented, and here are the notes from our review in the sound design meeting:
- We should try using a version of MSaBGlassLevelCrossfadeSound2 that is an octave lower for sound B (@Ashton-Morris referred to this as the "glockenspiel sound".
- @Ashton-Morris will also provide new sounds for both the A & B sounds that are a bit shorter, i.e. have a little less of a tail.
- We should use the term "cross fade" in the code to describe this.
- For the boundary sounds, use the same sound generator, but a bit more altered in pitch to emphasize the fact that a boundary has been reached.
- There are currently 7 middle crossing points.  All members of the team should play around with it and see if this feels right to them.
- Let's set the number of arrow key presses needed to traverse the full range to 8.  This will match up with the 7 crossing points.  As of this writing, it's 20.
- Do we need to use something other than linear values to mix the two sounds? We decided to leave as linear for now and try it again next week when we're further along and see if we need to use something different, like a "constant power" cross-fade.

**Comment by Ashton-Morris** [comment]
@jbphet  [I have uploaded these files as discussed.](https://o365coloradoedu.sharepoint.com/sites/PHYS-phet-pilot/PhET%20Files/Forms/AllItems.aspx?csf=1&web=1&e=jhc6X1&cid=83d3c58c%2D3b8d%2D4e50%2D8ea6%2D9748dc2b6762&RootFolder=%2Fsites%2FPHYS%2Dphet%2Dpilot%2FPhET%20Files%2FPhET%20Files%2FPhET%20Sim%20Design%2FMean%5F%20Share%20and%20Balance%20HTML5%2FSounds%20and%20Sonification%2FSound%20Files&FolderCTID=0x0120001AF96E22694A1A448A9649A49C4AC974)

- MSaBGlassLevelCrossfadeSound2OctaveLower.mp3
- MSaBGlassLevelCrossfadeSound2Shorter.mp3
- MSaBGlassLevelCrossfadeSound1Shorter.mp3

**Comment by jbphet** [comment]
Discussed at 4/2/2024 sound design meeting, but not all of the items listed above are implemented, just the new shorter sounds where one is an octave lower.  Next steps:

- Turn down the overall sound a bit so it's not as overwhelming
- Implement the other items on the list above

**Comment by jbphet** [comment]
Everything on the list above has been done, so this should be ready for the next round of review.  I'll leave it assigned to me so that I can explain what's been done during the sound design meeting.  I added one additional feature: I made the sounds into a major scale.  This seems a little more fun, especially since this sim is targeted at a relatively young audience.  We'll discuss this along with everything else at the next sound design meeting.
